### DISCUSSION TITLE
Possible to change reserved words?

### DISCUSSION THREAD

**Discussion by amrap030** [stalniy/casl#572] [Q&A]
Hello, I would like to know if it is possible to change the reserved word `manage` to a different one, so that the new word has the same behavior as the actual `manage` and `manage` will then work as any other word?

**Comment by stalniy** [comment]
Yes, it’s possible: https://github.com/stalniy/casl/blob/master/packages/casl-ability/src/RuleIndex.ts#L18

**Reply by amrap030** [reply]
Thank you for answering @stalniy , but I am still not sure how to use this feature. Could you give me a short code snippet, where to put what, to make it work?

Thanks in advance :)

**Reply by stalniy** [reply]
```js
const a = new Ability([
  { action: “*”, subject:”*” }
], {
 anyAction: “*”, 
anySubject: “*”
})

a.can(“*”, ”*”) // true
```

**Reply by amrap030** [reply]
Hmm doesn't work for me. I have the following code:

```js
import { Ability } from '@casl/ability';

export let ability = new Ability([], {
  anyAction: 'handle',
});

// in a different file
import { ability } from '@/frontend/helpers/iam/permissions/casl-abilities.js';

const rules = [
    {
        "action": [
            "manage",
            "view"
        ],
        "subject": "Process"
    },
    {
        "action": [
            "manage"
        ],
        "subject": "Process",
        "conditions": {
            "id": {
                "$in": [
                    "_dcc316cc-67a1-4b7a-929b-831e73b06f2a"
                ]
            }
        }
    }
]
ability.update(rules);
console.log(ability.can('share', 'Process')); // -> this results into true, but I expect it to be false, as I changed anyAction to "handle"
```

**Reply by stalniy** [reply]
Works in `@casl/ability@5.4.3`:

```ts
import { Ability } from "@casl/ability";

const ability = new Ability([], {
  anyAction: 'handle',
});

ability.update([{ action: "handle", subject: "Process"}]);
console.log(ability.can('read', 'Process')); // true
```

**Reply by amrap030** [reply]
Thank you @stalniy it works now. It was an issue with the version, I actually thought that I have installed the latest version. Now I have `anyAction` as `handle` and I would like to create an alias for `manage`, because it is not anymore working for `anyAction`. However, creating an alias is still not allowed, even if `anyAction` has changed.

**Reply by stalniy** [reply]
https://github.com/stalniy/casl/blob/master/packages/casl-ability/src/utils.ts#L111

`createAliasResolver` also accept options. You need to pass anyAction inside as well
